api/v1/runtimecomponent_types.go

```diff
@@ -146,6 +146,21 @@ type RuntimeComponentSpec struct {
 	// Security context for the application container.
 	// +operator-sdk:csv:customresourcedefinitions:order=25,type=spec,displayName="Security Context"
 	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
+
+	// Topology spread constraints for the application pod.
```

Comment:
This line can be removed, so the comment above RuntimeComponentTopologySpreadConstraints type's struct can be used

Reply:
Sounds good, I removed line 150.